### PR TITLE
Added pageUp and pageDown keys

### PR DESCRIPTION
**Pull request by unitehenry**
Added the ability to navigate slides with pageUp and pageDown as requested in issue #310

### CODE REVIEW

File: packages/components/src/MDXDeck.js [hunk 1]
```diff
@@ -90,10 +93,12 @@ export class MDXDeck extends React.Component {
       }
     } else {
       switch (keyCode) {
+        case keys.pgDown:
```
Comment (jxnblk) [inline packages/components/src/MDXDeck.js]: Thanks for this! I might be totally wrong, but shouldn't `pageDown` advance to the next slide?

Comment (unitehenry) [reply packages/components/src/MDXDeck.js]: Good question. I'll ask the author of the issue.

Comment (jxnblk) [reply packages/components/src/MDXDeck.js]: Based on what Google Slides does it seems to be page down for next slide

File: packages/components/src/MDXDeck.js [hunk 2]
```diff
@@ -35,6 +35,8 @@ const keys = {
   p: 80,
   o: 79,
   g: 71,
+  pgUp: 33,
```
Comment (vlsi) [inline packages/components/src/MDXDeck.js]: By the way, key codes are exactly the ones I've used during my presentation

**Comment by unitehenry** [comment]
Ok pushed the changes.

**Comment by jxnblk** [comment]
Thanks!

**Pull request merged**